.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+
+{
+	"name": "Codespaces for Thanos",
+
+	"context": "..",
+
+	"dockerFile": "",
+	
+	
+
+	"customizations": {
+		"vscode": {
+			"settings": { 
+				"go.toolsManagement.checkForUpdates": "local",
+				"go.useLanguageServer": true,
+				"go.gopath": "/go"
+			
+				}
+			}
+			
+			
+	},
+
+	"extensions": [
+		"ms-azuretools.vscode-docker",
+		"ms-kubernetes-tools.vscode-kubernetes-tools",
+		"davidanson.vscode-markdownlint",
+		"timonwong.shellcheck"
```

Comment:
Can you add the rest of the plugins? @Akshit42-hue

Reply:
Yes, will do it ASAP

Reply:
Hey @yeya24, Can you please give some pointers on where can I find all extensions mentioned in https://github.com/thanos-io/thanos/issues/5464#issuecomment-1204194870

Reply:
Probably just searching from vscode. Probably @douglascamata knows more

Reply:
As @yeya24, you should be able to find them just searching VSCode's Extension Marketplace.